src/lib/ws/server.ts

```diff
@@ -42,3 +47,17 @@ export const getWsServer = (): WsServer => {
 
 	return io;
 };
+
+export const fetchSockets = () => {
+	if (!io || !throttledFetchSockets) {
+		throw new Error('WS server not initialized yet');
+	}
+
+	const sockets = throttledFetchSockets();
+
+	if (sockets === undefined) {
+		throw new Error('undefined, the debounced function was not invoked yet');
```

Comment:
Don't quite see how this would happen. On the first call, it should be invoked right away.

Reply:
Me neither. But ts won't let me ignore that.

<img width="700" alt="image" src="https://user-images.githubusercontent.com/16905445/206188198-dc52756d-290f-4be9-a920-89201d056502.png">